lib/src/git_backend.rs

```diff
@@ -1259,6 +1289,8 @@ impl Backend for GitBackend {
                         source: Box::new(err),
                     })?;
 
+            // NOTE: I don't think this is needed in the world where we write
+            // change-id as a git header since the git id will be different.
```

Comment:
I think we also need to move the predecessors field out of the extras (or remove them completely)

Reply:
When you say this, do you mean removing them from the commit data structure entirely?